roles/webserver/tasks/main.yml

```diff
@@ -12,13 +12,20 @@
     - { src: "unicorn_init.j2", dest: "/etc/init.d/unicorn_{{ app }}", mode: "0744", owner: "{{ unicorn_user }}", group: "{{ unicorn_user }}" }
     - { src: "unicorn.rb.j2", dest: "{{ config_path }}/unicorn.rb", mode: "0744", owner: "{{ unicorn_user }}", group: "{{ unicorn_user }}" }
     - { src: "unicorn.service.j2", dest: "/etc/systemd/system/unicorn_{{ app }}.service", mode: "0744", owner: "{{ unicorn_user }}", group: "{{ unicorn_user }}" }
+    - { src: "delayed_job.service.j2", dest: "/etc/systemd/system/delayed_job_{{ app }}.service", mode: "0744", owner: "{{ unicorn_user }}", group: "{{ unicorn_user }}" }
   notify:
     - restart nginx
-  tags: unicorn
+  tags: init
 
-- name: update unicorn
+- name: Enable unicorn unit
   service:
     name: "unicorn_{{ app }}"
     enabled: yes
   become: yes
-  tags: unicorn
+  tags: init
+
+- name: Enable delayed_job unit
```

Comment:
It probably needs to become root just like in #144 or run as user service like in #143 
Depends on how we resolve #82

Reply:
Probably we'll need to become `root`

Reply:
With the current script for restarting it we run it as the unicorn user and it works fine...